### PR TITLE
#16 add SPDX compliant license, organization and scm information to the generated pom.xml

### PR DESCRIPTION
**Pull request by KemalSoysal**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -53,6 +53,38 @@ publishing {
...
+                        distribution = "repo"
+                    }
+                }
+                organization {
+                    name = "JetBrains s.r.o"
+                    url = "https://www.jetbrains.com"
+                }
+                scm {
+                    tag = "jb${jdk_version}-${jdk_build}"
+                    url = "https.//github.com/JetBrains/JetBrainsRuntime.git"
```
Comment (alexanderpann) [inline build.gradle:85]: typo after https

File: build.gradle [hunk 2]
```diff
@@ -53,6 +53,38 @@ publishing {
...
+                    // see https://spdx.org/licenses/ for list
+                    license {
+                        name = "GPL-2.0-only"
+                        url = "https://github.com/JetBrains/JetBrainsRuntime/blob/jb${jdk_version}-${jdk_build}/LICENSE"
+                        comments = "GNU GENERAL PUBLIC LICENSE Version 2, June 1991"
+                        distribution = "repo"
+                    }
+                    license {
+                        name = "WITH Classpath-exception-2.0"
+                        url = "https://github.com/JetBrains/JetBrainsRuntime/blob/jb${jdk_version}-${jdk_build}/ADDITIONAL_LICENSE"
```
Comment (alexanderpann) [inline build.gradle:68]: The file is called [ADDITIONAL\_LICENSE\_INFO](https://github.com/JetBrains/JetBrainsRuntime/blob/main/ADDITIONAL_LICENSE_INFO).

**Review by alexanderpann — CHANGES_REQUESTED** [review]
Looks good, just two minor typos.

File: build.gradle (hunk 1 above)
Comment (KemalSoysal) [reply build.gradle]: thank you for spotting the issue, fixed

File: build.gradle (hunk 2 above)
Comment (KemalSoysal) [reply build.gradle]: thank you for spotting the issue, fixed

**Review by alexanderpann — APPROVED** [review]
LGTM

**Pull request merged**